### DISCUSSION TITLE
Updated AS3 reference

### DISCUSSION THREAD

**Discussion by PippoApps** [airsdk/Adobe-Runtime-Support#856] [Q&A] [answered]
Is there some updated AS3 reference?
Adobe docs are stuck at AIR32, and I cannot find any reference for new classes and methods.
Cheers

**Comment by ajwfrost** [comment]
We're adding the updates to the end of the release notes: https://airsdk.harman.com/api/versions/33.1.1.476/release-notes/Release_Notes_AIR_SDK_33.1.1.476.pdf

I have a regular conversation with Adobe about taking over the AS3 documentation hosting, we're not yet quite sure how they do all the localisation, and we need to de-classify the asdoc mark-up (currently our legal team considers all the documentation as confidential since it's embedded in the source code... !). But, yes we should be getting this up properly before too long!

**Reply by PippoApps** [reply]
Thank you Andrew.
Looking forward :)

**Comment by ajwfrost** [comment, marked as answer]
Finally (thanks for your patience!) and with some help from @marchbold  ...

https://airsdk.dev/reference

**Reply by PippoApps** [reply]
Fantastic news!
Thank you!

**Reply by 2jfw** [reply]
Great to see the first throw on this 🍻
